libraries/AP_Scripting/applets/Script_Controller.lua

```diff
@@ -0,0 +1,195 @@
+--[[
+   a script to select other lua scripts using an auxillary switch
+--]]
+
+local THIS_SCRIPT = "Script_Controller.lua"
+local SEL_CH = 302
+local scripting_dir_select = assert(rc:find_channel_for_option(SEL_CH), 'no sw assigned to 302 for script directory selection') -- directory selection channel
```

Comment:
we don't need this with the aux cached method

Reply:
done